File: examples/weather/forecast/go/forecastWeather.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/weather"
)

// Get the weather forecast for the next 1-10 days
func ForecastWeather() {
	weatherService := weather.NewWeatherService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := weatherService.Forecast(&weather.ForecastRequest{
		Days:     2,
		Location: "London",
	})
	fmt.Println(rsp, err)
}

File: examples/weather/forecast/node/forecastWeather.js
const { WeatherService } = require("m3o/weather");

// Get the weather forecast for the next 1-10 days
async function forecastWeather() {
  let weatherService = new WeatherService(process.env.MICRO_API_TOKEN);
  let rsp = await weatherService.forecast({
    days: 2,
    location: "London",
  });
  console.log(rsp);
}

forecastWeather();
